packages/json/src/browser/json-client-contribution.ts

```diff
@@ -24,8 +24,11 @@ import {
     DocumentSelector
 } from '@theia/languages/lib/browser';
 import { JSON_LANGUAGE_ID, JSON_LANGUAGE_NAME, JSONC_LANGUAGE_ID } from '../common';
-import { ResourceProvider, DisposableCollection } from '@theia/core';
+import { ResourceProvider } from '@theia/core';
+import { DisposableCollection } from '@theia/core/lib/common';
 import URI from '@theia/core/lib/common/uri';
+import * as path from 'path';
```

Comment:
we should not pull node.js in the browser, there is `Path` in `@theia/core/lib/common/path`. https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#frontend-path